### PR TITLE
PowerFxV1 as default

### PR DESCRIPTION
**Pull request by anderson-joyle**
Issue https://github.com/microsoft/Power-Fx/issues/1472.

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by MikeStall — APPROVED** [review]

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs [hunk 1]
```diff
@@ -64,7 +64,7 @@ public sealed class Features
 
         internal bool PowerFxV1CompatibilityRules { get; set; }
 
-        internal static Features None => new Features();
+        public static Features None => new Features();
```
Comment (CarlosFigueiraMSFT) [inline src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs:67]: Does it need to be public? We had made it internal exactly to force people to use PowerFxV1 - and only let Power Apps take its time to catch up.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by CarlosFigueiraMSFT — DISMISSED** [review]
:shipit:

**Review by MikeStall — DISMISSED** [review]

**Review by CarlosFigueiraMSFT — APPROVED** [review]
:shipit:

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**